extensions/wikia/DesignSystem/scripts/DesignSystemGlobalNavigationSearchSuggestions.js

```diff
@@ -0,0 +1,86 @@
+require(
+	['jquery', 'wikia.window'],
+	function ($, window) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Function has a complexity of 15 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3AFunctionComplexity)